lib/datadog/lambda/trace/patch_http.rb

```diff
@@ -36,11 +36,7 @@ def request(req, body = nil, &block)
             Datadog::Trace.trace_context
           )
 
-          req[Datadog::Trace::DD_SAMPLING_PRIORITY_HEADER.to_sym] =
-            context[:sample_mode]
-          req[Datadog::Trace::DD_PARENT_ID_HEADER.to_sym] = context[:parent_id]
-          req[Datadog::Trace::DD_TRACE_ID_HEADER.to_sym] = context[:trace_id]
-          logger.debug("added context #{context} to request")
+          req = add_ctx_to_req(req, context)
```

Comment:
Return value here isn't req, and it doesn't look like req is used after this

Reply:
Req is used here in the call to super:

https://github.com/DataDog/datadog-lambda-rb/blob/aad8177265c2113e66340a07cbdf21a987e9cae6/lib/datadog/lambda/trace/patch_http.rb#L46

I've added the missing return statement - good catch.